pyproject.toml

```diff
@@ -35,6 +35,7 @@ mypy = "^0.782"
 pre-commit = "^2.6.0"
 pytest = "^6.0.1"
 pytest_mock = "^3.3.1"
+pandas = ">=0.20.0"
```

Comment:
if you don't mind, could you put this in alphabetical order?

Reply:
moving it to a custom "extras" block instead https://python-poetry.org/docs/pyproject/#extras